### PR TITLE
add dbnet yaml for synthtext dataset and td500 dataset

### PR DESCRIPTION
**Pull request by Songyuanwei**
Thank you for your contribution to the MindOCR repo.
Before submitting this PR, please make sure:

- [x] You have read the [Contributing Guidelines on pull requests](https://github.com/mindspore-lab/mindcv/blob/main/CONTRIBUTING.md)
- [x] Your code builds clean without any errors or warnings
- [x] You are using approved terminology
- [ ] You have added unit tests

## Motivation
add dbnet yaml for synthtext dataset and td500 dataset
(Write your motivation for proposed changes here.)

## Test Plan

(How should this PR be tested? Do you require special setup to run the test or repro the fixed bug?)

## Related Issues and PRs
 related issue #222 
(Is this PR part of a group of changes? Link the other relevant PRs and Issues here. Use https://help.github.com/en/articles/closing-issues-using-keywords for help on GitHub syntax)

### CODE REVIEW

File: configs/det/dbnet/README.md [hunk 1]
```diff
@@ -48,6 +57,17 @@ The overall architecture of DBNet is presented in _Figure 1._ It consists of mul
...
 
+### MSRA-TD500
+
+<div align="center">
+
+| **Model**         | **Context**    | **Backbone** | **Pretrained** | **Recall** | **Precision** | **F-score** | **Train T.** | **Throughput** | **Recipe**                  | **Download**                                                                                                                                                                                         |
+|-------------------|----------------|--------------|----------------|------------|---------------|-------------|--------------|----------------|-----------------------------|------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| DBNet (ours)      | D910x1-MS2.0-G | ResNet-50    | SynthText       | 82.47%     | 87.75%        | 85.03%      | 24.4 s/epoch  | 27.9 img/s      | [yaml](db_r50_td500.yaml) | [ckpt](https://download.mindspore.cn/toolkits/mindocr/dbnet/dbnet_resnet50_td500-0d12b5e8.ckpt)  |
```
Comment (SamitHuang) [inline configs/det/dbnet/README.md:66]: Why is the Throughput so different from SynthText with the same DBNet?  27.9 vs 82.02 FPS

File: configs/det/dbnet/README.md
Comment (SamitHuang) [inline configs/det/dbnet/README.md:1]: 需要在model readme中补充synthtext和td500的data preparation.

File: configs/det/dbnet/db_r50_synthtext.yaml
Comment (SamitHuang) [inline configs/det/dbnet/db_r50_synthtext.yaml:1]: 这里的weight decay 5e-4，跟原论文 1e-4并未对齐，原因是？（如调整后 最终loss更低？）

Comment (SamitHuang) [reply configs/det/dbnet/db_r50_synthtext.yaml:1]: 另外，net_columns_to_net为参数名，须对齐最新的：net_input_column_index,  label_column_index 设置上。

File: configs/det/dbnet/README.md [hunk 2]
```diff
@@ -34,6 +34,15 @@ The overall architecture of DBNet is presented in _Figure 1._ It consists of mul
 
 ## 2. Results
 
+### SynthText
+
+<div align="center">
+
+| **Model**         | **Context**    | **Backbone** | **Pretrained** |  **Train T.** | **Throughput** | **Recipe**                  | **Download**                 |
+|-------------------|----------------|--------------|----------------|------------|---------------|-------------|--------------|
+| DBNet (ours)      | D910x1-MS2.0-G | ResNet-50    | ImageNet       |  10470 s/epoch  | 82.02 img/s      | [yaml](db_r50_synthtext.yaml) | [ckpt](https://download.mindspore.cn/toolkits/mindocr/dbnet/dbnet_resnet50_synthtext-40655acb.ckpt)  |
```
Comment (SamitHuang) [inline configs/det/dbnet/README.md:43]: yaml中distribute为True，但此处D910x1-MS2.0-G 为单卡，please double-check standalone/distributed mode and the used number fo cards.

**Review by hadipash — APPROVED** [review]

File: configs/det/dbnet/db_r50_synthtext.yaml
Comment (hadipash) [reply configs/det/dbnet/db_r50_synthtext.yaml:1]: The original DBNet paper doesn't share details on the network pretraining with SynthText, it only mentions this:
> For all the models, we first pre-train them with the SynthText dataset for 100k iterations.

Nor the original repo has a config for SynthText. The "Synthetic Data for Text Localisation in Natural Images" paper has pretraining hyperparameters where they mentioned weight decay 5e-4 (actually, 5<sup>-4</sup> but I find it odd), however they used these hyperparameters to pretrain FCRN. So, I guess, it is up to us to find the best combination of hyperparameters to pretrain DBNet.

File: configs/det/dbnet/README.md
Comment (Songyuanwei) [reply configs/det/dbnet/README.md:1]: fixed

File: configs/det/dbnet/README.md (hunk 2 above)
Comment (Songyuanwei) [reply configs/det/dbnet/README.md:43]: fixed

Comment (SamitHuang) [reply configs/det/dbnet/README.md:43]: Please add training loss result for SynthText.

File: configs/det/dbnet/README.md (hunk 1 above)
Comment (Songyuanwei) [reply configs/det/dbnet/README.md]: batch_size is different. By adjusting num_workers, the current Throughput for TD500 is 51.1 img/s

File: configs/det/dbnet/README.md (hunk 2 above)
Comment (Songyuanwei) [reply configs/det/dbnet/README.md]: fixed

**Review by SamitHuang — APPROVED** [review]
